src/Exception/EntryDoesNotExistException.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
+namespace TomPHP\ConfigServiceProvider\Exception;
+
+final class EntryDoesNotExistException extends RuntimeException
```

Comment:
@localheinz This is actually referring to a local `RuntimeException` interface, something which I want to remove and replace with a single interface for all exceptions generated by this library - again, part of the review of exceptions.

With regards to importing root namespace classes, I've been pondering of this for a long time and never reached a decision. I'm happy to adopt whatever approach you prefer here?